tech_id.cpp

```diff
@@ -0,0 +1,9 @@
+#include "tech_id.h"
+
+const string getName(TechId id) {
+  return id.data();
+}
+
+const string getTechIdName(TechId id) {
```

Comment:
These are very stupid bootleg hacks which I don't know how to resolve. Perhaps I should use a namespace for these?

(without it there are conflicts between getName(contentFactory) and getName(TechId)